tools/iscc-sum/iscc_similarity.xml

```diff
@@ -0,0 +1,265 @@
+<tool id="iscc_sum_similarity" name="Find datasets with similar ISCC hashes" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="24.1">
+    <description>with ISCC-SUM</description>
+    <macros>
+        <import>macros.xml</import>
+        <import>creators.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <expand macro="version_command" />
+    <creator>
+        <expand macro="creators/iscc" />
+        <expand macro="creators/lco" />
+        <expand macro="creators/maartenpaul" />
+        <expand macro="creators/etzm" />
+    </creator>
+        
+    <command detect_errors="exit_code"><![CDATA[
+        ## Create working directory
+        mkdir -p input_files &&
+        
+        ## Create mapping file: filename -> element_identifier
+        echo "filename\telement_identifier" > id_mapping.tsv &&
+        
+        ## Symlink files and record their identifiers
+        #for $i, $file in enumerate($input_files):
+            ln -s '$file' 'input_files/${file.element_identifier}_${i}' &&
```

Comment:
the element_identifier should be unique, I don't think the `i` counter is needed here.